stubs/JACK-Client/jack/__init__.pyi

```diff
@@ -1,12 +1,38 @@
 import sys
 from _typeshed import Self
 from collections.abc import Callable, Generator, Iterable, Iterator, Sequence
-from typing import Any, overload
-from typing_extensions import Literal, TypeAlias
+from typing import Any, NoReturn, overload
+from typing_extensions import Literal
 
-_NDArray: TypeAlias = Any  # FIXME: no typings for numpy arrays
+import numpy
+from _cffi_backend import _CDataBase
+from numpy.typing import NDArray
 
-class _JackPositionT: ...
+# Aka jack_position_t
+# Actual type: _cffi_backend.__CDataOwn <cdata 'struct _jack_position *'>
+# This is not a real subclassing. Just ensuring type-checkers sees this type as compatible with _CDataBase
+# pyright has no error code for subclassing final
+class _JackPositionT(_CDataBase):  # type: ignore[misc]  # pyright: ignore
+    audio_frames_per_video_frame: float
+    bar: int
+    bar_start_tick: float
+    bbt_offset: int
+    beat: int
+    beat_type: float
+    beats_per_bar: float
+    beats_per_minute: float
+    frame: int
+    frame_rate: int
+    frame_time: float
+    next_time: float
+    padding: _CDataBase  # <cdata 'int32_t[7]'>
+    tick: int
+    ticks_per_beat: float
+    unique_1: int
+    unique_2: int
+    usecs: int
+    valid: Literal[0, 1]
```

Comment:
Are you sure about this? `position2dict` does bit fiddling on this field that suggests there are more possible values.

Reply:
Not 100% certain. I have no issues taking it safe and using `int`